### ISSUE TITLE
User Consent Breaks when Giver is not in Family.

### ISSUE THREAD

**Issue by jobannon** [FusionAuth/fusionauth-issues#1822]
### Description 
User Consent Breaks when Giver and consenter are not in same Family.  Freemarker error.

### Steps to reproduce
1. Create a Consent
2. Grant consent to a `userA` with a giver of `userB` (with `userB` being unrelated to `userA`, no family setup).
3. The consent is granted, but the UI does not respond on the users > your user > consents tab.

### Expected behavior
Consents tab to function.

### Screenshots
![image](https://user-images.githubusercontent.com/16090626/184245946-60e01d07-0621-4870-b99f-51858c1812f1.png)

### Platform
(Please complete the following information)
 - Device: [e.g. Desktop, iPhone X, Pixel 3, etc]
 - OS: [e.g. iOS, macOS, Windows ME]
 - Browser + version [e.g. chrome, safari plus a version]
 - Database [MySQL | PostgresSQL] & and version

### Community guidelines
All issues filed in this repository must abide by the [FusionAuth community guidelines](https://fusionauth.io/community/forum/topic/1000/code-of-conduct).

### Additional context
```
---
Tip: If the failing expression is known to legally refer to something that's sometimes null or missing, either specify a default value like myOptionalVar!myDefault, or use [#if myOptionalVar??]when-present[#else]when-missing[/#if]. (These only cover the last step of the expression; to cover the whole expression, use parenthesis: (myOptionalVar.foo)!myDefault, (myOptionalVar.foo)??
----

----
FTL stack trace ("~" means nesting-related):
	- Failed at: #assign giver = users(userConsent.giv...  [in template "templates/admin/user/_macros.ftl" in macro "consentTable" at line 787, column 13]
	- Reached through: @userMacros.consentTable  [in template "templates/admin/user/manage.ftl" at line 146, column 11]
	~ Reached through: #nested  [in template "templates/_utils/panel.ftl" in macro "full" at line 15, column 9]
	~ Reached through: @panel.full panelClass="panel ${panel...  [in template "templates/admin/user/manage.ftl" at line 96, column 7]
	~ Reached through: #nested  [in template "templates/_layouts/admin.ftl" in macro "main" at line 391, column 5]
	~ Reached through: @layout.main  [in template "templates/admin/user/manage.ftl" at line 82, column 5]
	~ Reached through: #nested  [in template "templates/_layouts/admin.ftl" in macro "body" at line 371, column 5]
	~ Reached through: @layout.body  [in template "templates/admin/user/manage.ftl" at line 38, column 3]
	~ Reached through: #nested  [in template "templates/_layouts/admin.ftl" in macro "html" at line 15, column 3]
	~ Reached through: @layout.html  [in template "templates/admin/user/manage.ftl" at line 28, column 1]
----
	at org.primeframework.mvc.freemarker.DefaultFreeMarkerService.render(DefaultFreeMarkerService.java:84)
	at org.primeframework.mvc.action.result.AbstractForwardResult.execute(AbstractForwardResult.java:92)
	at org.primeframework.mvc.action.result.DefaultResultInvocationWorkflow.perform(DefaultResultInvocationWorkflow.java:106)
	at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:50)
	at org.primeframework.mvc.scope.DefaultScopeStorageWorkflow.perform(DefaultScopeStorageWorkflow.java:61)
	at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:50)
	at org.primeframework.mvc.action.DefaultActionInvocationWorkflow.perform(DefaultActionInvocationWorkflow.java:64)
	at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:50)
	at org.primeframework.mvc.message.DefaultMessageWorkflow.perform(DefaultMessageWorkflow.java:50)
	at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:50)
	at org.primeframework.mvc.validation.DefaultValidationWorkflow.perform(DefaultValidationWorkflow.java:45)
	at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:50)
	at org.primeframework.mvc.security.DefaultSecurityWorkflow.perform(DefaultSecurityWorkflow.java:78)
	at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:50)
	at org.primeframework.mvc.parameter.DefaultPostParameterWorkflow.perform(DefaultPostParameterWorkflow.java:49)
	at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:50)
	at org.primeframework.mvc.content.DefaultContentWorkflow.perform(DefaultContentWorkflow.java:51)
	at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:50)
	at org.primeframework.mvc.parameter.DefaultParameterWorkflow.perform(DefaultParameterWorkflow.java:56)
	at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:50)
	at org.primeframework.mvc.parameter.DefaultURIParameterWorkflow.perform(DefaultURIParameterWorkflow.java:92)
	at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:50)
	at org.primeframework.mvc.scope.DefaultScopeRetrievalWorkflow.perform(DefaultScopeRetrievalWorkflow.java:50)
	at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:50)
	at org.primeframework.mvc.action.DefaultActionMappingWorkflow.perform(DefaultActionMappingWorkflow.java:110)
	at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:50)
	at org.primeframework.mvc.security.DefaultSavedRequestWorkflow.perform(DefaultSavedRequestWorkflow.java:66)
	at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:50)
	at org.primeframework.mvc.cors.CORSFilter.doFilter(CORSFilter.java:171)
	at org.primeframework.mvc.cors.CORSRequestWorkflow.perform(CORSRequestWorkflow.java:63)
	at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:50)
	at org.primeframework.mvc.workflow.DefaultMVCWorkflow.perform(DefaultMVCWorkflow.java:108)
	at org.primeframework.mvc.PrimeMVCRequestHandler.handleRequest(PrimeMVCRequestHandler.java:62)
	at org.primeframework.mvc.netty.PrimeHTTPServerHandler.channelRead0(PrimeHTTPServerHandler.java:158)
	at org.primeframework.mvc.netty.PrimeHTTPServerHandler.channelRead0(PrimeHTTPServerHandler.java:78)
	at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:99)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.netty.channel.ChannelInboundHandlerAdapter.channelRead(ChannelInboundHandlerAdapter.java:93)
	at io.netty.handler.codec.http.HttpServerKeepAliveHandler.channelRead(HttpServerKeepAliveHandler.java:64)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:286)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103)
	at io.netty.handler.codec.MessageToMessageCodec.channelRead(MessageToMessageCodec.java:111)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.netty.channel.CombinedChannelDuplexHandler$DelegatingChannelHandlerContext.fireChannelRead(CombinedChannelDuplexHandler.java:436)
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:327)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:299)
	at io.netty.channel.CombinedChannelDuplexHandler.channelRead(CombinedChannelDuplexHandler.java:251)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:722)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:658)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:584)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:496)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: freemarker.core.InvalidReferenceException: The following has evaluated to null or missing:
==> users(userConsent.giverUserId)  [in template "templates/admin/user/_macros.ftl" at line 787, column 30]

----
Tip: If the failing expression is known to legally refer to something that's sometimes null or missing, either specify a default value like myOptionalVar!myDefault, or use [#if myOptionalVar??]when-present[#else]when-missing[/#if]. (These only cover the last step of the expression; to cover the whole expression, use parenthesis: (myOptionalVar.foo)!myDefault, (myOptionalVar.foo)??
----

----
FTL stack trace ("~" means nesting-related):
	- Failed at: #assign giver = users(userConsent.giv...  [in template "templates/admin/user/_macros.ftl" in macro "consentTable" at line 787, column 13]
	- Reached through: @userMacros.consentTable  [in template "templates/admin/user/manage.ftl" at line 146, column 11]
	~ Reached through: #nested  [in template "templates/_utils/panel.ftl" in macro "full" at line 15, column 9]
	~ Reached through: @panel.full panelClass="panel ${panel...  [in template "templates/admin/user/manage.ftl" at line 96, column 7]
	~ Reached through: #nested  [in template "templates/_layouts/admin.ftl" in macro "main" at line 391, column 5]
	~ Reached through: @layout.main  [in template "templates/admin/user/manage.ftl" at line 82, column 5]
	~ Reached through: #nested  [in template "templates/_layouts/admin.ftl" in macro "body" at line 371, column 5]
	~ Reached through: @layout.body  [in template "templates/admin/user/manage.ftl" at line 38, column 3]
	~ Reached through: #nested  [in template "templates/_layouts/admin.ftl" in macro "html" at line 15, column 3]
	~ Reached through: @layout.html  [in template "templates/admin/user/manage.ftl" at line 28, column 1]
----
	at freemarker.core.InvalidReferenceException.getInstance(InvalidReferenceException.java:134)
	at freemarker.core.Assignment.accept(Assignment.java:139)
	at freemarker.core.Environment.visit(Environment.java:334)
	at freemarker.core.Environment.visit(Environment.java:376)
	at freemarker.core.IteratorBlock$IterationContext.executedNestedContentForCollOrSeqListing(IteratorBlock.java:291)
	at freemarker.core.IteratorBlock$IterationContext.executeNestedContent(IteratorBlock.java:271)
	at freemarker.core.IteratorBlock$IterationContext.accept(IteratorBlock.java:244)
	at freemarker.core.Environment.visitIteratorBlock(Environment.java:644)
	at freemarker.core.IteratorBlock.acceptWithResult(IteratorBlock.java:108)
	at freemarker.core.ListElseContainer.accept(ListElseContainer.java:40)
	at freemarker.core.Environment.visit(Environment.java:370)
	at freemarker.core.Environment.invokeMacroOrFunctionCommonPart(Environment.java:876)
	at freemarker.core.Environment.invokeMacro(Environment.java:812)
	at freemarker.core.UnifiedCall.accept(UnifiedCall.java:84)
	at freemarker.core.Environment.visit(Environment.java:370)
	at freemarker.core.Environment.invokeNestedContent(Environment.java:620)
	at freemarker.core.BodyInstruction.accept(BodyInstruction.java:60)
	at freemarker.core.Environment.visit(Environment.java:370)
	at freemarker.core.Environment.invokeMacroOrFunctionCommonPart(Environment.java:876)
	at freemarker.core.Environment.invokeMacro(Environment.java:812)
	at freemarker.core.UnifiedCall.accept(UnifiedCall.java:84)
	at freemarker.core.Environment.visit(Environment.java:370)
	at freemarker.core.Environment.invokeNestedContent(Environment.java:620)
	at freemarker.core.BodyInstruction.accept(BodyInstruction.java:60)
	at freemarker.core.Environment.visit(Environment.java:370)
	at freemarker.core.Environment.invokeMacroOrFunctionCommonPart(Environment.java:876)
	at freemarker.core.Environment.invokeMacro(Environment.java:812)
	at freemarker.core.UnifiedCall.accept(UnifiedCall.java:84)
	at freemarker.core.Environment.visit(Environment.java:370)
	at freemarker.core.Environment.invokeNestedContent(Environment.java:620)
	at freemarker.core.BodyInstruction.accept(BodyInstruction.java:60)
	at freemarker.core.Environment.visit(Environment.java:370)
	at freemarker.core.Environment.invokeMacroOrFunctionCommonPart(Environment.java:876)
	at freemarker.core.Environment.invokeMacro(Environment.java:812)
	at freemarker.core.UnifiedCall.accept(UnifiedCall.java:84)
	at freemarker.core.Environment.visit(Environment.java:370)
	at freemarker.core.Environment.invokeNestedContent(Environment.java:620)
	at freemarker.core.BodyInstruction.accept(BodyInstruction.java:60)
	at freemarker.core.Environment.visit(Environment.java:370)
	at freemarker.core.Environment.invokeMacroOrFunctionCommonPart(Environment.java:876)
	at freemarker.core.Environment.invokeMacro(Environment.java:812)
	at freemarker.core.UnifiedCall.accept(UnifiedCall.java:84)
	at freemarker.core.Environment.visit(Environment.java:334)
	at freemarker.core.Environment.visit(Environment.java:340)
	at freemarker.core.Environment.process(Environment.java:313)
	at freemarker.template.Template.process(Template.java:383)
	at org.primeframework.mvc.freemarker.DefaultFreeMarkerService.render(DefaultFreeMarkerService.java:76)
	... 72 common frames omitted

```

**Comment by LucasPMorris** [comment]
Related / Similar Issues: #722 #718

**Comment by andrewpai** [comment]
Closing, as this was fixed in 1.49.0.
